frontend/components/surplus/SurplusAuctionBidTransactionTable.vue

```diff
@@ -0,0 +1,145 @@
+            <div>Current Highest Bid</div>
+            <div>
+                <FormatCurrency :value="auction.bidAmountMKR" currency="MKR" />
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Lowest Next Bid</div>
+            <button class="ClickableText" :disabled="!isActive" @click="setTransactionBidAmount(undefined)">
+                <FormatCurrency :value="lowestNextBid" currency="MKR" />
+            </button>
+        </div>
+        <div class="flex justify-between items-center">
+            <div>The Amount to Bid</div>
+            <div class="flex w-1/2 items-center space-x-2 justify-end -mr-1">
+                <div class="w-full flex-shrink-0">
+                    <bid-input
+                        :transaction-bid-amount.sync="transactionBidAmount"
+                        :minimum-bid-dai="lowestNextBid"
+                        :disabled="!isActive"
+                        auction-type="surplus"
+                    />
+                </div>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Combined Transaction Fees</div>
+            <div>
+                <FormatCurrency v-if="auction.combinedBidFees" :value="auction.combinedBidFees" currency="DAI" />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Price on Uniswap</div>
+            <div>
+                <format-currency
+                    v-if="auction.marketUnitPrice && isActive"
+                    :value="auction.marketUnitPrice"
+                    currency="MKR"
+                />
+                <span v-else class="opacity-50">Unknown</span> per DAI
+            </div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>Price after the bid</div>
+            <div>
+                <format-currency
+                    v-if="unitPriceAfterBid && isActive && !isBidAmountNaN"
+                    :value="unitPriceAfterBid"
+                    currency="MKR"
+                />
+                <span v-else class="opacity-50">Unknown</span> per DAI
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import type { SurplusAuction } from 'auctions-core/src/types';
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import BidInput from '../utils/BidInput.vue';
+import TimeTill from '~/components/common/TimeTill.vue';
+import FormatCurrency from '~/components/utils/FormatCurrency.vue';
+
+export default Vue.extend({
+    components: {
+        TimeTill,
+        FormatCurrency,
+        BidInput,
+    },
+    props: {
+        auction: {
+            type: Object as Vue.PropType<SurplusAuction>,
+            required: true,
+        },
+        lowestNextBid: {
+            type: Object as Vue.PropType<BigNumber>,
+            required: true,
+        },
```

Comment:
This should be part of the auction, see here https://github.com/sidestream-tech/unified-auctions-ui/pull/345

Reply:
Yes, I am aware of this. @KirillDogadin-std was not sure when he would finish the PR, so we had agreed to initially calculate it here.

But seeing your new comments on the PR, I will change my types accordingly.

Reply:
I don't see the change yet!

Reply:
I don't see the change yet!

Reply:
I had changed it in the parent and provided it as a prop. But that makes no sense, so I just take it directly. My bad!